.circleci/config.yml

```diff
@@ -82,14 +82,17 @@ jobs:
           name: Install dependencies
           command: |
             HOMEBREW_NO_AUTO_UPDATE=1 brew install \
-              cmake \
+              wget \
               libyaml \
               cli11 \
               cairo
+            wget https://github.com/Kitware/CMake/releases/download/v3.31.9/cmake-3.31.9-macos-universal.tar.gz
+            tar xf cmake-3.31.9-macos-universal.tar.gz
```

Comment:
download and extract a good version of cmake

Reply:
In your PR #80, it seems the Mac build succeeds, and only the Windows build fails. So is it necessary to make this particular change? When installing libyaml from Homebrew, it may use the autoconf build system that's also there, rather than the CMake build system.